integration_tests/e2e/goal/createGoal.cy.ts

```diff
@@ -15,7 +15,22 @@ context('Create a goal', () => {
     cy.task('stubLearnerEducation')
   })
 
-  it('should not be able to navigate directly to Create Goal page given user has not clicked Add A Goal from overview page', () => {
+  it('should be able to navigate to Create Goal page given user has clicked Add A Goal from Overview page', () => {
```

Comment:
Confirm that existing behaviour of clicking Create Goal on PLP's overview screen still works